cra-tf-validate-ignore-rules.json

```diff
@@ -1,3 +1,16 @@
 {
-    "scc_rules": []
-}
+    "scc_rules": [
+      {
+        "scc_rule_id": "rule-9b2d8054-bc93-44fd-901b-91f677287e84",
+        "description": "Check whether Databases for PostgreSQL network access is restricted to a specific IP range",
```

Comment:
Have you run the SCC scan against this module? Based on the results you may need to update the rule id and description. If you need help running SCC scan you can reach out to me.

Reply:
Thanks @rajatagarwal-ibm I will reach out to you.